Robot.java

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+package frc.robot;
+
+import edu.wpi.first.wpilibj.Compressor;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.TimedRobot;
+
+/**
+* The VM is configured to automatically run this class, and to call the functions corresponding to
+* each mode, as described in the TimedRobot documentation. If you change the name of this class or
+* the package after creating this project, you must also update the build.gradle file in the
+* project.
+*/
+public class Robot extends TimedRobot {
+    private Compressor mCompressor;
+    private DoubleSolenoid mSolenoidOne;
+    private DoubleSolenoid mSolenoidTwo;
+
+    /**
+    * This function is run when the robot is first started up and should be used for any
+    * initialization code.
+    */
+    @Override
+    public void robotInit() {
+        mCompressor = new Compressor(0);
+        mCompressor.setClosedLoopControl(true);
+
+        mSolenoidOne = new DoubleSolenoid(1, 2);
+        mSolenoidOne = new DoubleSolenoid(1, 1, 2);
```

Comment:
Probably should be setting mSolenoidTwo, also we don't need to specify the module number. Also, I think we are using Single Solenoids